client/components/mma/MMAPage.tsx

```diff
@@ -625,7 +626,12 @@ const MMARouter = () => {
 												}
 											/>
 											<Route path="switch-thank-you" />
-											<Route path="confirm" />
+											<Route
```

Comment:
could add the lazy loading webpack chunking for this

Reply:
Good spot!